src/vs/workbench/contrib/search/common/searchModel.ts

```diff
@@ -368,9 +368,16 @@ export class FileMatch extends Disposable implements IFileMatch {
 		return Array.from(this._matches.values());
 	}
 
-	remove(match: Match): void {
-		this.removeMatch(match);
-		this._removedMatches.add(match.id());
+	remove(matches: Match | Match[]): void {
```

Comment:
changed this function to take an array; it doesn't directly impact this PR, but it might be handy in the future